src/parser/postgresparser.cpp

```diff
@@ -2061,6 +2064,69 @@ std::unique_ptr<VariableSetStatement> PostgresParser::VariableSetTransform(Parse
   return result;
 }
 
+<<<<<<< HEAD
```

Comment:
Think you may have left this in by accident

Reply:
Yup, just fixed that